Source/DafnyLanguageServer/Workspace/VerificationProgressReporter.cs

```diff
@@ -309,13 +308,40 @@ public class VerificationProgressReporter : IVerificationProgressReporter {
           targetMethodNode.StatusVerification = finalOutcome;
         }
 
+        var options = compilation.Program.Reporter.Options;
+        ReportVacuityAndRedundantAssumptionsChecks(compilation, implementation, verificationResult, options);
```

Comment:
Why not put this call here? https://github.com/dafny-lang/dafny/blob/master/Source/DafnyLanguageServer/Workspace/CompilationManager.cs#L402

I'd rather not add any code to `VerificationProgressReporter.cs`. I'm hoping to remove it after moving gutter icons to the client and updating hover so it doesn't need it.